### PR TITLE
[#XGB-63] Add managment summary

### PR DESCRIPTION
**Pull request by fabianhauser**
I tried to further simplify some statements from the abstract, I hope they are still accurate enough.

Did I cover all points we discussed @raphiz ?

### CODE REVIEW

**Comment by raphiz** [comment]
@fabianhauser great work, thanks! Have a look at my adoptions - we can discuss further details in person :wink: 
I have troube with the following sentence:
> In the future, the XMPP-Grid Broker can be extended by usability features like autocomplete or filtering, which could not be implemented efficiently due to missing features in the underlying XMPP standard.
It is rather confusing - because it says "we can do this in the future" and "we did not do it because it is not possible" at the same time... :smile:

File: documents/final-submission-document/final-submission-document.tex [hunk 1]
```diff
@@ -131,10 +131,37 @@ \chapter{Management Summary}
...
+A comprehensive analysis of the underlying standards is carried out in the first part of this thesis.
+Particular attention is given to portability, extensibility and security aspects in a production environment.
+By this analysis, a systematic selection of possible architecture options is executed in the form of architectural decisions.
+
+\section*{Results}
+
+The resulting implementation enables administrators to create and configure communication Topics, apprehend the hierarchy of the underlying XMPP-Grid and manage permissions of network participants.
+Additionally, persisted messages can be inspected and published on communication Topic.
+
+The XMPP-Grid Broker is implemented as Angular5 Web Application that connects directly from the web browser to the XMPP-Grid. Secure communication is assured by the use of Mutual authentication over TLS.
```
Comment (fabianhauser) [inline documents/final-submission-document/final-submission-document.tex]: I would have written web application lower-case, as it is an application in the web (which does not refer to a special concept of this thesis).

Would you like to keep it this way or should I change it?

Comment (fabianhauser) [reply documents/final-submission-document/final-submission-document.tex]: The same applies for "Mutual", but I assume this one is a typo?

**Comment by fabianhauser** [comment]
@raphiz thanks for the adoptions, looks largely great, I questioned the choice to capitalise some words as usual 😆

What do you think about:

> In the future, the XMPP-Grid Broker might be extended by usability features like autocomplete or filtering. Currently, these features could not be implemented efficiently due to missing support in the underlying XMPP standard, which would have to be extended and implemented by XMPP-Grids.

(We can discuss this in person or by ☎️ tomorrow if you like)

File: documents/final-submission-document/final-submission-document.tex (hunk 1 above)
Comment (raphiz) [reply documents/final-submission-document/final-submission-document.tex]: wopsy, Mutual is clearly wrong, and i agree with Web Application (which you wrote upper case in you first draft as well :wink: :laughing: ) :+1:

**Pull request merged**